documentation/OpenAPI/example.rst

```diff
@@ -0,0 +1,187 @@
+Example
+=======
+
+In this section, we will see a working example of OpenApi v3 client onto a simple API that gives facts about cats and comment it.
+
+.. warning::
+    We suggest you to read :doc:`/OpenAPI/generate` and :doc:`/OpenAPI/usage` first to understand more easily.
+
+You can find the fully working example on the following link: https://github.com/janephp/openapi3-example
+
+OpenAPI schema
+--------------
+
+First, we need a valid OpenAPI schema. You can use tool such as Stoplight_ or other OpenApi designer.
+
+I choosed to represent `CatFacts API`_ within this example. So here is the quick schema I made
+
+.. code-block:: yaml
+
+    openapi: 3.0.0
+    info:
+        version: 1.0.0
+        title: 'CatFacts API'
+    paths:
+        /facts/random:
+            get:
+                operationId: randomFact
+                responses:
+                    200:
+                        description: 'Get a random `Fact`'
+                        content:
+                            application/json:
+                                schema:
+                                    $ref: '#/components/schemas/Fact'
+    components:
+        schemas:
+            Fact:
+                type: object
+                properties:
+                    _id:
+                        type: string
+                        description: 'Unique ID for the `Fact`'
+                    __v:
+                        type: integer
+                        description: 'Version number of the `Fact`'
+                    user:
+                        type: string
+                        description: 'ID of the `User` who added the `Fact`'
+                    text:
+                        type: string
+                        description: 'The `Fact` itself'
+                    updatedAt:
+                        type: string
+                        description: 'Date in which `Fact` was last modified'
+                    sendDate:
+                        type: string
+                        description: 'If the `Fact` is meant for one time use, this is the date that it is used'
+                    deleted:
+                        type: boolean
+                        description: 'Weather or not the `Fact` has been deleted (Soft deletes are used)'
+                    source:
+                        type: string
+                        description: 'Can be `user` or `api`, indicates who added the fact to the DB'
+                    used:
+                        type: boolean
+                        description: 'Weather or not the `Fact` has been sent by the CatBot. This value is reset each time every `Fact` is used'
+                    type:
+                        type: string
+                        description: 'Type of animal the `Fact` describes (e.g. ‘cat’, ‘dog’, ‘horse’)'
+
+This will allow us to request CatFact API and get a random Fact !
```

Comment:
```suggestion
This will allow us to request CatFact API and get a random Fact!
```

I think this whole phrase could be dropped. Or rephrase:

> This schema describe the endpoint and the model of the CatFact API.

Reply:
```suggestion
This schema describe the endpoint and the model of the CatFact API.
```